src/pointer-analysis/value_set_domain_transform.inc

```diff
@@ -25,13 +26,18 @@ void value_set_domaint::transform(
     break;
 
   case END_FUNCTION:
-    value_set.do_end_function(static_analysis_baset::get_return_lhs(to_l), ns);
+  {
+    value_set.do_end_function(
+      static_analysis_baset::get_return_lhs(to_l), ns);
```

Comment:
No actual changes here?

Reply:
Yeah, the existing code spilled over the 80-column boundary so I got linter complaints on moving the file like this.